<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QFrame" name="frameMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>980</width>
     <height>780</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeWidget" name="treeWidgetModules">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>175</width>
      <height>740</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Категории</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelModuleName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>520</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите модуль...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>750</width>
      <height>700</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>748</width>
       <height>698</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelConstDescription">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Описание</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelModuleDescription">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>70</y>
        <width>700</width>
        <height>450</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>660</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать демонстрацию</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxParameters">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>710</width>
        <height>105</height>
       </rect>
      </property>
      <property name="title">
       <string>Параметры</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
